components/navbar/navbar.jsx

```diff
@@ -355,6 +365,17 @@ export default class Navbar extends React.Component {
         );
     }
 
+    handleUnmuteChannel = () => {
```

Comment:
We could probably make a view action to make this a bit simpler if you wanted since the component doesn't really need to know how actually unmute the channel. Something like
```javascript
function unmuteChannel(channelId, mute) {
    return (dispatch, getState) => {
        const currentUserId = getCurrentUserId(getState());

        dispatch(updateChannelNotifyProps(currentUserId, channelId, {
            mark_unread: mute ? NotificationLevels.MENTIONS : NotificationLevels.ALL,
        });
    };
}
```

Reply:
Definitely better but I suggest to have it on separate PR as improvement if you agree.